docs/integrations-guides/eigenda-proxy.md

```diff
@@ -0,0 +1,36 @@
+# EigenDA Proxy
+
+## About
+EigenDA proxy is a sidecar server ran as part of a rollup node cluster for communication with the EigenDA network. 
```

Comment:
```suggestion
EigenDA proxy is a sidecar REST server meant to be run next to a rollup node for communicating with the EigenDA network. 
```

Reply:
"cluster" was confusing to me, and probably will be to users. If this is meant to be an intro guide, then we should just say node<1-1>proxy

Reply:
[4d3d60d](https://github.com/Layr-Labs/eigenda-docs/pull/66/commits/4d3d60d271bd334a04f90f8f9361387be9523a75)